src/rmkit/fb/stb_text.cpy

```diff
@@ -141,7 +141,8 @@ namespace stbtext:
     for j = 0; j < image.h; j++:
       for i = 0; i < image.w; i++:
         uint32_t val = text_buffer[j*image.w+i]
-        image.buffer[j*image.w+i] = val == 0 ? WHITE: BLACK;
+        //image.buffer[j*image.w+i] = val == 0 ? WHITE: BLACK;
```

Comment:
Remove instead of commenting out?

Reply:
I left it in case someone else wanted the old behavior back, but I'm fine to remove it

Reply:
Isn't that usually that's what git history, using an older version of the library, or adding a feature flag is for?